pkg/hintrunner/zero/zerohint_memcpy.go

```diff
@@ -3,9 +3,54 @@ package zero
 import (
 	"github.com/NethermindEth/cairo-vm-go/pkg/hintrunner/core"
 	"github.com/NethermindEth/cairo-vm-go/pkg/hintrunner/hinter"
+	"github.com/NethermindEth/cairo-vm-go/pkg/utils"
+	"github.com/NethermindEth/cairo-vm-go/pkg/vm/memory"
+	"github.com/consensys/gnark-crypto/ecc/stark-curve/fp"
+
 	VM "github.com/NethermindEth/cairo-vm-go/pkg/vm"
 )
 
+func newMemcpyContinueCopyingHinter(output hinter.ResOperander) hinter.Hinter {
+	return &GenericZeroHinter{
+		Name: "MemcpyContinueCopying",
+		Op: func(vm *VM.VirtualMachine, ctx *hinter.HintRunnerContext) error {
+			//> n -= 1
+			//> ids.continue_copying = 1 if n > 0 else 0
+			var lhs fp.Element
+
+			//> n-=1
+			n, err := ctx.ScopeManager.GetVariableValue("n")
+			if err != nil {
+				return err
+			}
+
+			ctx.ScopeManager.AssignVariable("n", lhs.Sub(n.(*fp.Element), &utils.FeltOne))
+
+			//> ids.continue_copying = 1 if n > 0 else 0
+			continueCopyingAddr, err := output.GetAddress(vm)
+			if err != nil {
+				return err
+			}
+
+			var v memory.MemoryValue
+			if !utils.FeltLt(&utils.FeltZero, n) {
```

Comment:
```
if utils.FeltLt(&utils.FeltZero, n) {
      v = memory.MemoryValueFromFieldElement(&utils.FeltOne)
} else {
      v = memory.MemoryValueFromFieldElement(&utils.FeltZero)
}
```

this removes the `!` operator and respect the order of the python hint (strictly no importance though ^^)

Reply:
fixed now! ✅

Reply:
thank you! @TAdev0